packages/rancher-tracing/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
-packageVersion: 01
+packageVersion: 02
```

Comment:
@brendarearden can you confirm if this is the first time we’re updating rancher-tracing for this release?

Reply:
correct - there have been no changes for rancher-tracing chart for 2.5.6. The changes that were needed for tracing were corrected from the istio chart. good call out!